Cargo.lock

```diff
@@ -1088,7 +1088,7 @@ dependencies = [
  "semver",
  "serde",
  "serfig",
- "thrift",
+ "thrift 0.18.0",
```

Comment:
It looks like this means that there are two versions of thrift at the same time, and I think that thrift 0.18 works, but what about the other older version?

Reply:
thrift specify the version to `v0.18` after releasing `v0.17`.

Reply:
> but what about the other older version?

`opentelemetry-jaeger` depends on `v0.15`, we don't know if we can patch this too.